### DISCUSSION TITLE
Who is using MapLibre Native?

### DISCUSSION THREAD

**Discussion by louwers** [maplibre/maplibre-native#690] [General]
Based on a sparring session with @artakka: we should find a few (ideally large) apps that use MapLibre Native and document these users somewhere.

Not only is it good marketing, it will also help us make sure new releases are stable. When doing a release, it is good when our tests pass and our dummy test apps still work. However the best test is if real apps running on real devices are still functional.

Please share here if you are using MapLibre Native or the users that you know of. It would be helpful to know if the creators of these apps are willing to be featured and if they are willing to try out pre-releases.

**Comment by birkskyum** [comment]
https://github.com/maplibre/maplibre-gl-native/issues/379#issuecomment-1250268426

- [Delta Chat](https://github.com/deltachat/deltachat-android) - [issue](https://github.com/deltachat/deltachat-android/issues/2361), [fix-by-updating](https://github.com/deltachat/deltachat-android/pull/2351), [f-droid](https://f-droid.org/en/packages/com.b44t.messenger/)
- [Transportr](https://github.com/grote/Transportr) - [issue](https://github.com/grote/Transportr/issues/668), [f-droid](https://f-droid.org/en/packages/de.grobox.liberario/)
- [Element](https://github.com/vector-im/element-android) - [issue](https://github.com/vector-im/element-android/issues/6100), [fix-by-excluding](https://github.com/vector-im/element-android/pull/6136), [f-droid](https://f-droid.org/en/packages/im.vector.app/)
- [EVMap](https://github.com/johan12345/EVMap) - [issue](https://github.com/johan12345/EVMap/issues/197), [fix-by-exclusion-using-product-flavour](https://github.com/johan12345/EVMap/blob/1f36ef6af8d2134b25695b25b46391e9158380d1/app/build.gradle)

**Reply by louwers** [reply]
https://ev-map.app/

**Comment by roblabs** [comment]
A couple of Open Source projects that consume the MapLibre SDK via Swift Package Manager:   [AWS Location Samples](https://github.com/aws-samples/amazon-location-samples/tree/main/maplibre-native-ios) & [Open Map Tiles Demo](https://github.com/roblabs/openmaptiles-ios-demo#build).

**Comment by thehoneymad** [comment]
Amazon Location, for sure. Need to look into others.

**Comment by boldtrn** [comment]
[Kurviger, motorcycle route planning and navigation app](https://play.google.com/store/apps/details?id=com.kurviger.app)

**Comment by louwers** [comment]
I reached out to Moopmoop, a weather aware navigation app. https://moopmoop.nl/

They are using MapLibre Native.

**Comment by louwers** [comment]
Liberty Rider is using MapLibre Native. https://liberty-rider.com/

**Comment by louwers** [comment]
Geovelo is using MapLibre Native https://geovelo.app

**Comment by louwers** [comment]
Entur is using MapLibre Native for some of their developer tools. https://entur.no/

**Comment by louwers** [comment]
https://www.graphmasters.net/ is using MapLibre Native.

**Comment by louwers** [comment]
[MapMyIndia](https://www.mapmyindia.com/) is using MapLibre Native.

**Comment by manuelroth** [comment]
Swiss Federal Railways is using [MapLibre Native](https://www.sbb.ch/en/travel-information/apps/sbb-mobile.html) and [MapLibre GL JS on the web](https://tinyurl.com/5u3n67d4) for their travel companion product

**Comment by manuelroth** [comment]
Komoot is using [MapLibre Native](https://www.komoot.com/) and [MapLibre GL JS](https://www.komoot.com/plan/) for their route planning product

**Comment by Helium314** [comment]
[StreetComplete](https://github.com/streetcomplete/StreetComplete) is in the process of switching to MapLibre Native. I hope we'll be done in a month or two.

**Reply by riQQ** [reply]
Starting from [v59.0](https://github.com/streetcomplete/StreetComplete/releases/tag/v59.0) StreetComplete is using MapLibre Native.

**Comment by louwers** [comment]
[RouteYou](https://www.routeyou.com/) is using MapLibre Native.

**Comment by Sideness** [comment]
Mappy is using Maplibre Native for [iOS](https://apps.apple.com/fr/app/mappy-gps-trafic-prix/id313834655) (and soon [Android](https://play.google.com/store/apps/details?id=com.mappy.app)) and [Web](https://fr.mappy.com/)

**Reply by louwers** [reply]
Thanks for sharing. 👍

**Comment by manuelroth** [comment]
Swiss Radio and Television SRF is using MapLibre for its weather app on [iOS](https://apps.apple.com/ch/app/srf-meteo-wetter-schweiz/id887941884), [Android](https://play.google.com/store/apps/details?id=ch.srf.meteo) and [Web](https://www.srf.ch/meteo/radar)

**Comment by DonTomika** [comment]
[menetrend.app](https://menetrend.app/) (a popular public transit app in Hungary) is using MapLibre Native on [Android](https://play.google.com/store/apps/details?id=hu.donmade.menetrend.budapest).

**Comment by louwers** [comment]
[Topo Lock Maps](https://topolock.com/blog/ios-app-now-available/) is using MapLibre Native.

Gorgeous app by the way @voncannon!

**Reply by voncannon** [reply]
Hey I really appreciate that coming from you! 😄

**Comment by hactar** [comment]
I'm involved in the following projects that use MapLibre:
- Bikemap cycling navigation: https://www.bikemap.net
- The official City of Vienna app: https://apps.apple.com/at/app/stadt-wien/id1014988960?
- Upcoming navigation app: https://hudhud.sa/en (unreleased)

**Comment by louwers** [comment]
The official Wikipedia app for Android uses MapLibre Native. https://github.com/wikimedia/apps-android-wikipedia

**Comment by jwinarske** [comment]
Planned for Automotive Grade Linux (AGL) fall release.  AGL is a Linux Foundation project.

**Comment by louwers** [comment]
[Threema](https://threema.ch/en) is using MapLibre Native for [Android](https://github.com/search?q=repo%3Athreema-ch%2Fthreema-android%20maplibre&type=code).

**Comment by louwers** [comment]
[IKEA](https://www.ikea.com/) is using MapLibre Native for one of its apps.

**Comment by kaushalbx** [comment]
Ola maps is using Maplibre.
Ola Maps is a navigation platform developed by the Indian ride-hailing company Ola (biggest rival of UBER in India), tailored specifically for India's dynamic road conditions.
In July 2024, Ola replaced Google Maps with its in-house Ola Maps across its platforms and announced an API for developers, allowing integration of its mapping services into third-party applications.

**Reply by louwers** [reply]
Thanks for sharing!

**Reply by birkskyum** [reply]
@kaushalbx , do you know if the usage of MapLibre is MapLibre Native, MapLibre GL JS (web) or both?

**Reply by kaushalbx** [reply]
MapLibre GL JS – used in web applications with Ola Maps (JavaScript/React).

MapLibre Native – used in their mobile SDKs (Android/iOS) for native rendering.


So the answer is: both.

**Comment by louwers** [comment]
I think MapLibre Native is used by Netcetera maybe @Stanislava99 can confirm. 🙂 

https://apps.apple.com/lu/app/corporate-mobility/id6468036118
https://play.google.com/store/apps/details?id=com.netcetera.corporatemobility.android.app

**Reply by Stanislava99** [reply]
Yes, that's right, it is used by Netcetera mobile team
https://play.google.com/store/apps/details?id=ch.blt.mobile.android&hl=en&pli=1
https://apps.apple.com/ch/app/basel-go/id582674328?l=en-GB

**Reply by louwers** [reply]
Very cool!

**Comment by ianthetechie** [comment]
I'll leave a shout for @Archdoog's [Rallista](https://rallista.app/), [Villages GPS](https://villagesgps.com/), and [Navigate PTC](https://play.google.com/store/apps/details?id=com.peachtreecity&pli=1).

**Comment by Archdoog** [comment]
> I'll leave a shout for @Archdoog's [Rallista](https://rallista.app/), [Villages GPS](https://villagesgps.com/), and [Navigate PTC](https://play.google.com/store/apps/details?id=com.peachtreecity&pli=1).

Our next generation apps that aren't released yet 😅

**Comment by MubarakNative** [comment]
[MBCompass](https://github.com/MubarakNative/MbCompass/) is using MapLibre for showing user current location (although this project is evolving) it is nice to mention foss app here.

**Comment by systemed** [comment]
[cycle.travel](https://cycle.travel/app) uses MapLibre Native for both Android and iOS.

**Comment by Buthrakaur** [comment]
[SmartGuide](https://www.smartguide.app/) travel guide app uses MapLibre via https://github.com/maplibre/maplibre-react-native for both Android and iOS

**Reply by louwers** [reply]
Thanks for sharing!

**Comment by cachly** [comment]
My app [Cachly](https://www.cachly.com) is using MapLibre native iOS for offline maps.

**Comment by louwers** [comment]
https://www.apollopharmacy.in/ is using MapLibre for their medicine delivery app.

**Comment by louwers** [comment]
[OCELL](https://www.ocell.io/en-us) is using MapLibre Flutter now according to @ AlexanderThiele. 🙌

**Comment by louwers** [comment]
The apps of the Germany-wide [Stadtradeln](https://www.stadtradeln.de/home) project use MapLibre!
![B478ACB6-3A52-4EA1-AE04-FC73CAC48C50_1_105_c](https://github.com/user-attachments/assets/471bb079-978e-4593-a241-ad70e2b81a87)

**Comment by louwers** [comment]
/e/OS is using MapLibre Native https://e.foundation/e-os/

![image](https://github.com/user-attachments/assets/19647dfd-4b6f-4cf7-be9a-7352339d423a)

**Comment by louwers** [comment]
Rivian is using MapLibre! https://www.linkedin.com/posts/colincren_electric-vehicle-charging-stations-rivian-activity-7351656136560816129-t7v9

https://rivian.com/experience/charging

<img width="629" height="516" alt="image" src="https://github.com/user-attachments/assets/01627cc6-1c2f-48af-bc3b-5af5163d211b" />

**Comment by halset** [comment]
The mostly Norwegian map app [NaVida](https://apps.apple.com/no/app/navida/id356821974?l=nb) that I am working on.
![IMG_C99F50D60541-1](https://github.com/user-attachments/assets/ed62b059-cd4b-4ce8-aa10-5c96987db191)

**Reply by louwers** [reply]
Takk for sharing!

**Comment by louwers** [comment]
I think [Gojek](https://www.gojek.com/en-id) is using MapLibre Native on iOS and Android.

Is this true @radityagumay?

**Reply by radityagumay** [reply]
@louwers yes that's correct. we recently using Maplibre for the Gojek Driver App Android 
:)

**Comment by sirlaurie** [comment]
I used MapLibre Native on my personal iOS app [Path](https://apps.apple.com/app/id6758724528), it's Moves like app, still actively under development

<img width="603" height="1311" alt="image" src="https://github.com/user-attachments/assets/d7505c71-b65e-4ed0-a4cd-9d89895ac7c4" />

**Reply by louwers** [reply]
Thanks for sharing! Looks great already. :)

**Comment by serialrf433** [comment]
Columba is currently a Android app. In the future there is also a iOS version planned.
Current app description: Native Android messaging app using Bluetooth LE, TCP, or RNode (LoRa) over LXMF and Reticulum 
It is using MapLibre: https://github.com/torlando-tech/columba

**Comment by davwheat** [comment]
[TrainSplit](https://trainsplit.app/), a railway ticketing retailer for Great Britain, uses MapLibre Native in both our Android and iOS apps.

Currently, the use is limited to display polygons for where bus add-on tickets (called [PlusBus](https://www.plusbus.info/birmingham)) is valid, but we're currently working on railway journey maps too using our own custom vector map tiles.

<img width="2076" height="2152" alt="image" src="https://github.com/user-attachments/assets/75bfa6cc-44cc-49f5-a66f-0fb6521ad998" />

**Reply by louwers** [reply]
Thanks for sharing!
